tests/postgresql_configuration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
move the db creation to the setup per test,
for example:  tests/testenv/__init__.py
include: create + cleanup